### ISSUE TITLE
Pissing On / Burning Down the Corporation platform thingy

### ISSUE THREAD

**Issue by Pfhorrest** [Pfhorrest/Eternal#203]
Aaron wanted something changed about this platform, I forget what now (delay before activation?)

![Screen Shot 2020-04-02 at 12 12 57 AM](https://user-images.githubusercontent.com/5233345/78229875-6ad46c00-7485-11ea-9300-1f15b345b599.png)

**Comment by aaronfreed** [comment]
IIRC, it needs "can't deactivate externally" set.

**Comment by aaronfreed** [comment]
Fixed most of these issues; just not sure how to fix the “stuck compiler” thing. (Might just have to make all those walls solid.) On the other hand, there was also an issue with a S’pht’Kr getting stuck on the northwestern circular platform and screwing up its timing; I simply raised the ceiling of that by 1 WU to eliminate the problem. Also added a bit of fusion ammo to the areas with the drones in each of these levels, to give the player a reward for exploring those areas. The one in “Burning” is a secret anyway so it really doesn’t matter whether the ammo makes story sense.

[Pissing-on-the-Corporation.sceA.zip](https://github.com/Pfhorrest/Eternal/files/4464357/Pissing-on-the-Corporation.sceA.zip)

[Burning-Down-The-Corporation.sceA.zip](https://github.com/Pfhorrest/Eternal/files/4464356/Burning-Down-The-Corporation.sceA.zip)
